extensions/MyDashboard/web/styles/mydashboard.css

```diff
@@ -71,3 +70,17 @@
 #last_changes .field_label {
     text-align: left;
 }
+
+#saved_searches_container > header {
+    display: flex;
+    align-items: center;
+    margin: 5px 0;
+}
+
+#saved_searches_container > header h2 {
+    margin: 0 10px 0 0;
+}
+
+#saved_searches_container li {
+    margin: 5px 0;
```

Comment:
I have over 30 saved searches so a vertical layout isn't great for displaying them. Can we use a horizontal layout more like the old footer so I don't have a large blank area causing more scrolling on the page

Reply:
Maybe a responsive grid layout?

Reply:
That sounds like overkill for a temporary thing. The following would be good enough IMO:
```css
#saved_searches_container li {
    margin: 5px;
    display: inline;
}
```
![image](https://user-images.githubusercontent.com/601139/37065843-898ff8d6-2157-11e8-9ce0-bac396ccae17.png)

Reply:
Done in #451. Needs a bug filed.